universe/Species.cpp

```diff
@@ -30,6 +30,15 @@ FocusType::FocusType(std::string& name, std::string& description,
     m_graphic(std::move(graphic))
 {}
 
+FocusType::FocusType(std::string&& name, std::string&& description,
```

Comment:
-this is not a move constructor (that would have signature `FocusType(FocusType&& rhs)`
-it also seems to duplicate the one right above, so I'm confused why there isn't a compile error

Reply:
I was not sure now to call it.

Reply:
it's a constructor taking all parameters by rvalue reference I guess?

Reply:
Yes.